packages/ckeditor5-image/ckeditor5-metadata.json

```
@@ -249,7 +249,14 @@
 					"type": "SplitButton",
 					"name": "insertImage",
 					"iconPath": "@ckeditor/ckeditor5-core/theme/icons/image.svg"
-				},
+				}
+			]
+		},
+		{
+			"name": "Image Insert via Url",
```

```suggestion
			"name": "Image insert via URL",
```

Or even rather "Insert image via URL", but I see above that there's "Image insert", so :shrug: